server/sonar-server/src/main/java/org/sonar/server/component/DefaultRubyComponentService.java

```diff
@@ -39,6 +39,7 @@
 
 import static org.sonar.server.component.NewComponent.newComponentBuilder;
 
+// TODO delete this class
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)